composer.json

```diff
@@ -29,28 +29,28 @@
         "snc/redis-bundle": "^2.1",
         "stof/doctrine-extensions-bundle": "^1.3",
         "sylius/mailer-bundle": "^1.3",
-        "symfony/asset": "^4.1",
-        "symfony/console": "^4.1",
-        "symfony/expression-language": "^4.1",
-        "symfony/flex": "^1.0",
-        "symfony/framework-bundle": "^4.1",
-        "symfony/http-foundation": "^4.1",
+        "symfony/asset": "^4.2",
+        "symfony/console": "^4.2",
+        "symfony/expression-language": "^4.2",
+        "symfony/flex": "^1.2",
+        "symfony/framework-bundle": "^4.2",
+        "symfony/http-foundation": "^4.2",
         "symfony/lts": "^4@dev",
```

Comment:
Why we use the dev tag version?

Reply:
Removed cause package has this description > " Abandoned! See symfony/flex"